docs/reaction.md

```diff
@@ -63,11 +61,11 @@ And this simplifies the use of spherical harmonics $Y_l^m(\theta,\phi)$,
 since only the orbital angular momentum component is involved, and thus the combination of contribution is not considered (e.g. Clebsch-Gordan Coefficients).
 
 In our case:
+
 - $A^{12}$ amplitude represents a d-wave interaction, as indicated by $l=2$.
-    - The possible $m$ values are $−2,−1,0,1,2$. Each of these values corresponds to different orientations of the d-wave. The wave type is solely determined by $l$ and all these $m$ values still describe d-wave characteristics.
+  - The possible $m$ values are $−2,−1,0,1,2$. Each of these values corresponds to different orientations of the d-wave. The wave type is solely determined by $l$ and all these $m$ values still describe d-wave characteristics.
 - $A^{23}$ amplitude represents a p-wave interaction, as indicated by $l=1$.
-    - The possible $m$ values are $−1,0,1$. Each of these values corresponds to different orientations of the p-wave. Similarly, these values are all p-wave orientations.
-- $A^{31}$ amplitude represents a s-wave interaction, as indicated by $l=0$.
-    - The only possible $m$ value is 0, which is consistent with the spherical symmetry of s-waves.
+  - The possible $m$ values are $−1,0,1$. Each of these values corresponds to different orientations of the p-wave. Similarly, these values are all p-wave orientations.
+- $A^{31}$ amplitude represents a s-wave interaction, as indicated by $l=0$. - The only possible $m$ value is 0, which is consistent with the spherical symmetry of s-waves.
```

Comment:
Prettier is messing up some things here. But this is usually a sign that the indentation of the bullet points was not consistent. Double check this.